From 3291a728c8919cab933ff3637f28e0180e316be9 Mon Sep 17 00:00:00 2001
From: ZaneZam <cyxman@yahoo.com>
Date: Sun, 1 Feb 2015 18:07:41 +0100
Subject: [PATCH] init.rc patch

---
 init.rc | 22 +++++++++++++---------
 1 file changed, 13 insertions(+), 9 deletions(-)

diff --git a/init.rc b/init.rc
index c2398c9..6151e5b 100755
--- a/init.rc
+++ b/init.rc
@@ -58,9 +58,9 @@ loglevel 3
 
     # See storage config details at http://source.android.com/tech/storage/
     mkdir /mnt/shell 0750 shell shell
-    mkdir /mnt/media_rw 0700 media_rw media_rw
+    mkdir /mnt/media_rw 0750 media_rw media_rw
     mkdir /storage 0751 root sdcard_r
-        
+
     # Directory for putting things only root should see.
     mkdir /mnt/secure 0700 root root
     # Create private mountpoint so we can MS_MOVE from staging
@@ -136,6 +136,10 @@ loglevel 3
 # This is needed by any process that uses socket tagging.
     chmod 0644 /dev/xt_qtaguid
 
+# ko files for exfat -> AP: modules instead compiled into kernel
+    insmod /lib/modules/exfat_core.ko
+    insmod /lib/modules/exfat_fs.ko
+
 on fs
 # mount mtd partitions
     # Mount /system rw first to give the filesystem a chance to save a checkpoint
@@ -763,13 +767,6 @@ service powersnd /system/bin/samsungpowersoundplay
      disabled
      oneshot
 
-service bootanim /system/bin/bootanimation
-    class main
-    user graphics
-    group graphics system
-    disabled
-    oneshot
-
 service installd /system/bin/installd
     class main
     socket installd stream 600 system system
@@ -883,3 +880,10 @@ service pppoe /system/bin/pppoewrapper
     disabled
     oneshot
 # PPPOE SUPPORT
+
+# Call Boeffla-Kernel init script
+service boeffla-init /sbin/boeffla-init.sh
+    class main
+    user root
+    group root
+    oneshot
-- 
2.1.4

